modulefiles/module_base.orion_bk

```diff
@@ -0,0 +1,68 @@
+#%Module######################################################################
```

Comment:
This file is not needed, is there an example test case on orion since the orion modules were updated?

Reply:
Should this be removed?

Reply:
This file needs to be removed before PR is merged.